README.md

```diff
@@ -1,5 +1,5 @@
 # Microsoft Graph .NET Authentication Provider Library
-[![Build status](https://o365exchange.visualstudio.com/O365%20Sandbox/_apis/build/status/Microsoft%20Graph/PR-msgraph-validate-sdk-dotnet-auth)](https://o365exchange.visualstudio.com/O365%20Sandbox/_build/latest?definitionId=0)
+[![Build status](https://o365exchange.visualstudio.com/O365%20Sandbox/_apis/build/status/Microsoft Graph/.Net/msgraph-sdk-dotnet-auth-build-and-packaging)](https://o365exchange.visualstudio.com/O365%20Sandbox/_build/latest?definitionId=2425)
```

Comment:
You'll need to url encode the space in `Microsoft Graph/.Net/msgraph-sdk-dotnet-auth-build-and-packaging` with a `%20`.